scripts/process_data.py

```diff
@@ -971,9 +1045,128 @@ def main(self) -> None:
         CONSOLE.rule()
 
 
+@dataclass
+class ProcessPolycam:
+    """Process Polycam data into a nerfstudio dataset.
+
+    To capture data, use the Polycam app on an iPhone or iPad with LiDAR. The capture must be in LiDAR or ROOM mode.
+    Developer mode must be enabled in the app settings, this will enable a raw data export option in the export menus.
+    The exported data folder is used as the input to this script.
+
+    This script does the following:
+
+    1. Scales images to a specified size.
+    2. Converts Polycam poses into the nerfstudio format.
+    """
+
+    data: Path
+    """Path the polycam export data folder. Can be .zip file or folder."""
+    output_dir: Path
+    """Path to the output directory."""
+    num_downscales: int = 3
+    """Number of times to downscale the images. Downscales by 2 each time. For example a value of 3
+        will downscale the images by 2x, 4x, and 8x."""
+    use_uncorrected_images: bool = False
+    """If True, use the raw images from the polycam export. If False, use the corrected images."""
+    max_dataset_size: int = 300
+    """Max number of images to train on. If the dataset has more, images will be sampled approximately evenly. If -1,
+    use all images."""
+    min_blur_score: float = 25
+    """Minimum blur score to use an image. If the blur score is below this value, the image will be skipped."""
+    crop_border_pixels: int = 30
+    """Number of pixels to crop from the border of the image. Useful as borders may be black due to undistortion."""
+
+    verbose: bool = False
+    """If True, print extra logging."""
+
+    def main(self) -> None:
+        """Process images into a nerfstudio dataset."""
+
+        self.output_dir.mkdir(parents=True, exist_ok=True)
+        image_dir = self.output_dir / "images"
+        image_dir.mkdir(parents=True, exist_ok=True)
+
+        summary_log = []
+
+        if self.data.suffix == ".zip":
+            with zipfile.ZipFile(self.data, "r") as zip_ref:
+                zip_ref.extractall(self.output_dir)
+                extracted_folder = zip_ref.namelist()[0].split("/")[0]
+            self.data = self.output_dir / extracted_folder
+            print(self.data)
+
+        if self.use_uncorrected_images:
+            polycam_image_dir = self.data / "keyframes" / "images"
+            polycam_cameras_dir = self.data / "keyframes" / "cameras"
```

Comment:
crop_border_pixels should be 0 in this case also. In fact, I'd suggest cleaning these lines up by doing 
```
if (self.data / "keyframes" / "corrected_images").exists() and not use_uncorrected_images:
   # use corrected images
else:
  crop_border_pixels = 0
   # use raw images
```